### PR TITLE
GUVNOR-3365: test marshalling of tree structure

### PR DESCRIPTION
**Pull request by jomarko**
@tarilabs If you don't mind, please have a look.

### CODE REVIEW

File: kie-wb-common-dmn/kie-wb-common-dmn-backend/src/test/java/org/kie/workbench/common/dmn/backend/DMNMarshallerTest.java [hunk 1]
```diff
@@ -299,6 +305,60 @@ public void roundTripUnmarshalThenMarshalUnmarshal(InputStream dmnXmlInputStream
         checkGraphConsumer.accept(g2);
     }
 
+    private void checkTreeStructureGraph(Graph<?, Node<?, ?>> graph) {
```
Comment (tarilabs) [inline kie-wb-common-dmn/kie-wb-common-dmn-backend/src/test/java/org/kie/workbench/common/dmn/backend/DMNMarshallerTest.java]: Thank you for this additional test. Having several tests is never a bad idea, just not getting what is on high level the difference between this new test and the pre-existing `potpourri` test, please? Can you clarify this?

Comment (jomarko) [inline kie-wb-common-dmn/kie-wb-common-dmn-backend/src/test/java/org/kie/workbench/common/dmn/backend/DMNMarshallerTest.java]: @tarilabs there is probably no difference on the high level (content of the file). I just wanted to use also file created directly by the new DMN editor.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@manstis I would ask you for merging.

**Pull request merged**